src/IOST/Signer.cpp

```diff
@@ -0,0 +1,131 @@
+#include "Signer.h"
+#include "Account.h"
+#include "../Hash.h"
+#include "../PrivateKey.h"
+#include <boost/endian/conversion.hpp>
+#include <sstream>
+#include <iostream>
+
+#include <nlohmann/json.hpp>
+
+using namespace TW;
+using namespace TW::IOST;
+
+class IOSTEncoder {
+  public:
+    IOSTEncoder() {}
```

Comment:
/wallet-core/src/IOST/Signer.cpp:16:5: warning: use '= default' to define a trivial default constructor [modernize-use-equals-default]